### PR TITLE
Update changelogs

### PR DESCRIPTION
**Pull request by liuche**
IS CHANGELOG CHANGES

Question: in the future, do we want to add LATs to the changelog too?
* Would make it easier to not get out of date, if we have multiple LATs (though changelogs will still be added to the releases page)
* Would need to duplicate changelog entries, because a LAT is not a release (and those features would get released officially later)

### Pull Request checklist

- [ ] This PR includes thorough **tests** or an explanation of why it does not
- [X] This PR includes a **CHANGELOG entry** or does not need one

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,14 +7,25 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 
 ### Added
+- Add feature to cast from YouTube mobile app (on phone, tablet)
```
Comment (mcomella) [inline CHANGELOG.md]: nit: - Feature to cast...

Add is redundant

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,14 +7,25 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
 
 ### Changed
+- Hide Pocket video tile on non-EN locales
+    - Pocket does not yet provide content for other languages
+
```
Comment (mcomella) [inline CHANGELOG.md]: nit: rm newline

File: CHANGELOG.md [hunk 3]
```diff
@@ -45,6 +56,9 @@ play/pause/rewind/fast-forward/restart/next/previous"
 ## Note on releases below
 The CHANGELOG entries for the releases listed below this were added retroactively and may be incomplete.
 
+## [2.1.2] 2018-10-30
+Version-bump only for *Fire TV Stick 4k*
```
Comment (mcomella) [inline CHANGELOG.md]: nit: I've been italicizing any content that doesn't fit into this pattern so I'd italicize this whole line.

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,14 +7,25 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
 ### Fixed
 - Blank screen when pressing back from a full-screened video
 - Infrequent crash caused by initialization logic (#1159)
 
+## [3.0.2] - 2018-10-30
+Version-bump only, released adding *Stick Gen 1 & 2* in addition to *Fire TV (Gen 1, 2, 3), Cube, Element 4k (pendant)*
+
+## [3.0.1] - RC candidate, did not release
+### Fixed
+Issue that would cause browsing history to be lost when navigating to device home screen
```
Comment (mcomella) [inline CHANGELOG.md]: nit: Add "-" before

File: CHANGELOG.md [hunk 5]
```diff
@@ -7,14 +7,25 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+    - Pocket does not yet provide content for other languages
+
 - Sentry crash reports include a UUID to distinguish users so we can determine if it's 1 user crashing 100 times or 100 users crashing 1 time each. This identifier is only used for Sentry and can not be correlated with telemetry interaction data. See [fire TV Sentry docs](https://github.com/mozilla-mobile/firefox-tv/wiki/Crash-reporting-with-Sentry) for more details. (#817)
 
 ### Fixed
 - Blank screen when pressing back from a full-screened video
 - Infrequent crash caused by initialization logic (#1159)
 
+## [3.0.2] - 2018-10-30
+Version-bump only, released adding *Stick Gen 1 & 2* in addition to *Fire TV (Gen 1, 2, 3), Cube, Element 4k (pendant)*
```
Comment (mcomella) [inline CHANGELOG.md]: nit: italicize whole line

Comment (mcomella) [inline CHANGELOG.md]: I'm not sure what "released adding ..." means. Do you mean, "Released v3.0+ for the first time to Stick Gen 1 & 2 in addition to (devices) that already had v3.0+"?

File: CHANGELOG.md (hunk 4 above)
Comment (mcomella) [inline CHANGELOG.md]: nit: I try to add issue numbers to fixed items, e.g. - Issue that causes regressions (#450000)

**Comment by liuche** [comment]
Fixed nits, thanks for pointing them out!

**Pull request merged**